tests/eas/st_hold.py

```diff
@@ -0,0 +1,380 @@
+#    Copyright 2018-2018 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pandas as pd
+from env import TestEnv
```

Comment:
Nit: I don't think we have an official style for imports, but the [holy book](https://www.python.org/dev/peps/pep-0008/#imports) states that it should be standard modules first, then third party modules, then local modules.

Reply:
ok, I'm updating this